delta_shooter/shooter_utils.py

```diff
@@ -0,0 +1,65 @@
+import random
+from pathlib import Path
+
+from pygame import Color
+from pygame.image import load
+from pygame.math import Vector2
+from pygame.mixer import Sound
+from pygame.surface import Surface
+
+HERE = Path(__file__).parent.resolve()
+
+
+def load_sprite(name: str, with_alpha: bool = True) -> Surface:
+    path = HERE / f"assets/sprites/{name}.png"
+    loaded_sprite = load(path)
+
+    if with_alpha:
+        return loaded_sprite.convert_alpha()
+    else:
+        return loaded_sprite.convert()
+
+
+def load_sound(name: str) -> Sound:
+    path = HERE / f"assets/sounds/{name}.wav"
+    return Sound(str(path))
+
+
+def wrap_position(position, surface):
```

Comment:
Pls could you type the rest of this file?

Reply:
Guessing its not because it's a copy-paste job

Reply:
it sure is a copy-paste job 👯‍♀️